dipy/viz/tests/test_apps.py

```diff
@@ -105,6 +105,7 @@ def test_horizon(rng):
     streamlines.append(s1)
     streamlines.append(s2)
     streamlines.append(s3)
+    streamlines.shrink_data()
```

Comment:
IMO Deserves a separate commit, since it looks like this is the change that will fix #3231, the rest of changes being STYLE changes.

Reply:
done in 3bbdeccbf3010b0c9b44f547e6c3fa5ac8d9b8f4

Reply:
> the rest of changes being STYLE changes.

It was not style since both were fixing warnings generated by the new FURY release and keyword only arguments rules

Reply:
see here https://github.com/dipy/dipy/actions/runs/10199389702/job/28216359136#step:11:4208

Reply:
> It was not style since both were fixing warnings generated by the new FURY release and keyword only arguments rules

OK for FIX then :+1:. Thanks for having created a separate commit for the issue with the array.